packages/mobile/src/app/App.tsx

```diff
@@ -64,7 +64,13 @@ const App = () => {
   useEffectOnce(() => {
     setLibs(null)
     subscribeToNetworkStatusUpdates()
-    TrackPlayer.setupPlayer({ autoHandleInterruptions: true })
+    TrackPlayer.setupPlayer({
+      minBuffer: 0.1,
+      playBuffer: 0.1,
+      waitForBuffer: false,
```

Comment:
nice additions, this actually change things from what you can tell?

Reply:
hard to tell in dev mode and tbh not really but they are "correct" to some degree for what we want the library to do